vl-proza-message.src.js

```diff
@@ -154,4 +138,146 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
     }
 }
 
+/**
+ * VlProzaMessagePreloader
+ * @class
+ * @classdesc
+ *
+ * @extends VlElement
+ *
+ * @property {string} data-vl-domain - Het Proza domein waarin de Proza berichten zitten.
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-proza-message/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-proza-message/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-proza-message.html|Demo}
+ *
+ */
+export class VlProzaMessagePreloader extends VlElement(HTMLElement) {
+    static get _observedAttributes() {
+        return ['data-vl-domain'];
+    }
+
+    constructor() {
+        super();
+        this._preload();
+    }
+
+    _data_vl_domainChangedCallback() {
+        this._preload();
+    }
+
+    get _domain() {
+        return this.dataset.vlDomain;
+    }
+
+    _preload() {
+        if (this._domain) {
+            VlProzaMessagePreloader._preload(this._domain);
+        }
+    }
+
+    static _preload(domain) {
+        if (!VlProzaMessagePreloader.isPreloaded(domain)) {
+            VlProzaMessagePreloader.__setPreloadedMessagesCacheForDomain(domain, ProzaRestClient.getMessages(domain));
+        }
+    }
+
+    /**
+     * Geeft een Proza bericht terug.
+     *
+     * @param domain - Het Proza domein waarin het Proza bericht zit.
+     * @param code - De code die het Proza bericht identificeert.
+     * @returns {Promise<string>} Resolved naar het Proza bericht indien teruggevonden en anders wordt de Promise rejected.
```

Comment:
Triviaal: '-' vergeten.

Reply:
Had gekeken naar de docs van returns maar zag daar geen optie om een '-' te zetten zoals bij param. Misschien komt dat vanwege het feit dat er bij param nog een naam van de parameter voor de beschrijving moet komen, en bij returns niet. Hier kan dus geen verwarring zijn waarvoor '-' nodig is.